tests/Feature/Products/FamiliesControllerTest.php

```
@@ -286,12 +286,12 @@ public function it_deletes_family(): void
 
         /* Act */
         $response = $this->actingAs($user)->post(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$user'. (undefined)

(UnusedLocalVariable)